usaspending_api/api_contracts/search_filters.md

```diff
@@ -177,6 +177,30 @@ Request parameter description:
 * `date_type`:  (enum[string])
     Check specific time period objects for date_type's members, defaults, and descriptions.
 
+## Award Search Time Period Object
+
+**Description:**
+See [Time Period](#time-period)
+
+**Examples**
+See [Time Period](#time-period)
+
+Request parameter description:
++ `start_date`: (required)
+    See [Time Period](#time-period)
++ `end_date`: (required)
+    See [Time Period](#time-period)
++ `date_type`: (optional)
+    + Members
+        + `action_date`
+            This date type value is the default type compared to the `start_date`.
+        + `date_signed`
+            This date type value is the default type compared to the `end_date`.
+        + `last_modified_date`
+        + `new_awards_only`
+            Indicates when the results should reflect new awards only. You should expect only
+            awards whose base transaction date falls within the time period bounds specified to be returned.
```

Comment:
I think it might make more sense to say "earliest transaction" here instead of "base transaction"

Reply:
Andrew Ly and Ben call it base transaction. I used the terminology "base transaction" to be consistent with them. Does that change your mind?

Reply:
This is fine if that's the language they like to use